lib/meilisearch-rails.rb

```diff
@@ -545,7 +545,12 @@ def ms_index!(document, synchronous = false)
             end
           end
         end
-        nil
+
+        if tasks.count <= 1
+          tasks.first
```

Comment:
I don't like the approach of returning multiple types here. As a developer consuming the method, I would always have to check the response to use the response. So I suggest returning the tasks directly.

Reply:
But better thinking about it, I don't know if there is a use case to return multiple tasks. Can you check that?

Reply:
As agreed by Discord, we will make this return a single type an Array